minifi/minifi-nar-bundles/minifi-framework-bundle/minifi-server-nar/pom.xml

```diff
@@ -43,6 +43,12 @@
             <artifactId>nifi-headless-server</artifactId>
             <version>1.17.0-SNAPSHOT</version>
         </dependency>
+        <dependency>
+            <groupId>org.apache.commons</groupId>
+            <artifactId>commons-lang3</artifactId>
+            <version>${org.apache.commons.lang3.version}</version>
```

Comment:
This version can be removed since it is defined in the root Maven configuration.